twoliter/embedded/Makefile.toml

```diff
@@ -16,6 +16,8 @@ BUILDSYS_TOOLS_DIR = "${BUILDSYS_ROOT_DIR}/tools"
 BUILDSYS_SOURCES_DIR = "${BUILDSYS_ROOT_DIR}/sources"
 BUILDSYS_SBKEYS_DIR = "${BUILDSYS_ROOT_DIR}/sbkeys"
 BUILDSYS_SBKEYS_PROFILE = { script = ['echo "${BUILDSYS_SBKEYS_PROFILE:-local}"'] }
+BUILDSYS_CACERTS_DIR = "${BUILDSYS_ROOT_DIR}/certs"
+BUILDSYS_CACERTS_BUNDLE = { script = ['echo "${BUILDSYS_CACERTS_BUNDLE:-${BUILDSYS_CACERTS_DIR}/ca-bundle.crt}"'] }
```

Comment:
Per my feedback in `buildsys` - I don't think this should be unconditionally set to a real path.